sympy/functions/elementary/complexes.py

```diff
@@ -466,7 +466,7 @@ def eval(cls, arg):
                     if base.func is cls and exponent is S.NegativeOne:
                         return arg
                     return Abs(base)**exponent
-                if base.is_positive == True:
+                if base.is_positive:
```

Comment:
+1 too. It's just needlessly complicated, and will make people think that this is the way to do comparisons.